modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -429,13 +429,6 @@ input FluxDensityContinuumSurfaceInput {
   units: FluxDensityContinuumSurfaceUnits!
 }
 
-# Focal plane angle source angle in appropriate units
-input FocalPlaneAngleInput {
-  microarcseconds: Long
-  milliarcseconds: BigDecimal
-  arcseconds: BigDecimal
-}
-
```

Comment:
I deleted this in favor of using generic `AngleInput` everywhere for consistency.